src/passes/Precompute.cpp

```diff
@@ -79,7 +79,7 @@ class PrecomputingExpressionRunner
     if (iter != getValues.end()) {
       auto values = iter->second;
       if (values.isConcrete()) {
-        return Flow(std::move(values));
+        return Flow(values);
```

Comment:
The `std::move` I removed here actually existed already before the offending PR, which I guess is valid as long as `getValues` is guaranteed to live longer than the runner does? Shall I add it back?

Reply:
Nah, `SmallVector` doesn't even have a move constructor so the `std::move` here never did anything anyway. IIRC I put it in there in case we ever got around to adding a move constructor in the future. But if we did that we might want to do a general audit of where it should be used, so removing it now sgtm.